src/legacy/core_plugins/interpreter/types/index.ts

```diff
@@ -17,14 +17,4 @@
  * under the License.
  */
 
-export { ArgumentType } from './arguments';
-export {
-  TypeToString,
-  KnownTypeToString,
-  TypeString,
-  UnmappedTypeStrings,
-  UnwrapPromise,
-} from './common';
-export { ExpressionFunction } from './functions';
-export { ExpressionType } from './types';
-export * from '../common/types';
+export * from '../../../../plugins/data/common/expressions/types/index';
```

Comment:
@joshdover is it alright that we’re importing from deep in the plugin here? I suppose this is just a temporary thing for migration

Reply:
> ... is it alright ...

I assume it is not. But those are just TS types and the plan is to delete this file. I can make it a top level export now, if you would like.

Reply:
Changed, importing from top level now.